### PR TITLE
Fix broken image links in DevTools release notes

### PR DESCRIPTION
**Pull request by kenzieschmoll**
This will fix https://github.com/flutter/devtools/issues/5767.

### CODE REVIEW

File: src/tools/devtools/release-notes/release-notes-2.23.1-src.md
Comment (parlough) [inline src/tools/devtools/release-notes/release-notes-2.23.1-src.md:10]:
```suggestion
![verbose logging]({{site.url}}/tools/devtools/release-notes/images-2.23.1/verbose-logging.png "verbose_logging")
```

File: src/tools/devtools/release-notes/release-notes-2.23.1-src.md
Comment (parlough) [inline src/tools/devtools/release-notes/release-notes-2.23.1-src.md:26]:
```suggestion
![method table]({{site.url}}/tools/devtools/release-notes/images-2.23.1/cpu-method-table.png "method_table")
```

**Review by parlough — COMMENTED** [review]
Thanks for catching and fixing these.

We've broken up the `/development` directory, so have to adjust these a bit:

We'll have to wait until after the freeze ends soon to merge.

**Comment by kenzieschmoll** [comment]
Why does /development need to be added? I don't see `development` in the path to these images and src files? https://github.com/flutter/website/tree/main/src/tools/devtools/release-notes

If that is the case, all the links in the previous DevTools release notes will be broken too since they don't reference `development`. However, the previous release notes images don't seem broken on the website: https://docs.flutter.dev/tools/devtools/release-notes/release-notes-2.20.0

**Comment by parlough** [comment]
`/development` doesn't need to be added, in those review suggestions I'm removing it.

**Comment by kenzieschmoll** [comment]
Oh whoops, I must have been on an old branch when I copied the file path. Fixed

**Review by parlough — APPROVED** [review]
Thanks!

**Pull request merged**
